rqt_controller_manager/rqt_controller_manager/controller_manager.py

```diff
@@ -254,8 +255,10 @@ def _on_ctrl_menu(self, pos):
         elif ctrl.state in ("finalized", "inactive"):
             if action is action_activate:
                 self._activate_controller(ctrl.name)
-            elif action is action_unload:
+            elif action is action_cleanup:
+                self._deactivate_controller(ctrl.name)
                 unload_controller(self._node, self._cm_name, ctrl.name)
+                load_controller(self._node, self._cm_name, ctrl.name)
```

Comment:
```suggestion
            elif action is action_cleanup:
                # TODO: use cleanup service once available
                unload_controller(self._node, self._cm_name, ctrl.name)
                load_controller(self._node, self._cm_name, ctrl.name)
```
why calling `deactivate` here, if we are already in `inactive`?

Reply:
I had the entire chain in mind for some reason, from active to unconfigured, but since this is only for inactive, you're correct, makes no sense to have it here